tests/conftest.py

```
@@ -5,216 +5,141 @@
-            # Try to connect to server
-            response = requests.get(
-                "http://localhost:8000",
-                timeout=2
-            )
-            # Accept any response code - just need to confirm server is running
+            response = requests.get("http://127.0.0.1:8000", timeout=2)
             if response.status_code in [200, 304]:
                 server_ready = True
-                elapsed = time.time() - start_time
-                print(f"\r✅ Server ready after {elapsed:.1f}s")
+                print(f" ✅ Ready!")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)